<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(25, 11, 11);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Test Grila &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;INFORMATICA&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTimeEdit" name="timeEdit">
       <property name="displayFormat">
        <string>HH:mm:ss</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>491</width>
      <height>281</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_name">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>251</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(171, 236, 170);</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Nume si Prenume:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_START">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>150</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(19, 234, 204);
color: rgb(15, 8, 8);</string>
      </property>
      <property name="text">
       <string>START</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLabel" name="questionLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>459</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(18, 235, 216);</string>
      </property>
      <property name="text">
       <string>questionLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_NEXT">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>210</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(21, 221, 67);</string>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Back">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>210</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(74, 118, 89);
</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>461</width>
        <height>108</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="answer1">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(128, 164, 239);</string>
         </property>
         <property name="text">
          <string>answer1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="answer2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(241, 141, 234);
</string>
         </property>
         <property name="text">
          <string>answer2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="answer3">
         <property name="styleSheet">
          <string notr="true">
background-color: rgb(147, 241, 159);</string>
         </property>
         <property name="text">
          <string>answer3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="answer4">
         <property name="styleSheet">
          <string notr="true">
background-color: rgb(207, 135, 240);</string>
         </property>
         <property name="text">
          <string>answer4</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*color: rgb(255, 255, 255);*/
color: rgb(31, 53, 224);</string>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>599</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
